wordpress-mobile/WordPress-iOS
# Use the Tracks Sentry integration

Integrate Sentry using Tracks

**To Test:**
- Add these lines to a file and run the app in a simulator:

```swift
import AutomatticTracks

CrashLogging.logMessage("This is a test!")
```

Run the app in a way that causes the code above to run, and verify that it shows up in Sentry.
